packages/editor-ui/src/views/SettingsLdapView.vue

```diff
@@ -0,0 +1,793 @@
+			tableKey: 0,
+			adConfig: {} as ILdapConfig,
+			loadingTestConnection: false,
+			loadingDryRun: false,
+			loadingLiveRun: false,
+			loadingTable: false,
+			hasAnyChanges: false,
+			formInputs: null as null | IFormInputs,
+			formBus: new Vue(),
+			readyToSubmit: false,
+			page: 0,
+			loginEnabled: false,
+			syncEnabled: false,
+		};
+	},
+	async mounted() {
+		await this.getLdapConfig();
+	},
+	computed: {
+		...mapStores(useUsersStore, useSettingsStore),
+		currentUser(): null | IUser {
+			return this.usersStore.currentUser;
+		},
+		isLDAPFeatureEnabled(): boolean {
+			return this.settingsStore.settings.enterprise.ldap === true;
+		},
+	},
+	methods: {
+		openDocsPage(event: MouseEvent): void {
+			window.open('https://n8n.io/pricing/', '_blank');
+		},
+		cellClassStyle({ row, column }: { row: rowType; column: cellType }) {
+			if (column.property === 'status') {
+				if (row.status === 'Success') {
+					return { color: 'green' };
+				} else if (row.status === 'Error') {
+					return { color: 'red' };
+				}
+			}
+			if (column.property === 'runMode') {
+				if (row.runMode === 'Dry') {
+					return { color: 'orange' };
+				} else if (row.runMode === 'Live') {
+					return { color: 'blue' };
+				}
+			}
+			return {};
+		},
+		onInput(input: { name: string, value: string | number | boolean }) {
+			if (input.name === 'loginEnabled' && typeof input.value === 'boolean') {
+				this.loginEnabled = input.value;
+			}
+			if (input.name === 'synchronizationEnabled' && typeof input.value === 'boolean') {
+				this.syncEnabled = input.value;
+			}
+			this.hasAnyChanges = true;
+		},
+		onReadyToSubmit(ready: boolean) {
+			this.readyToSubmit = ready;
+		},
+		syncDataMapper(sync: ILdapSyncData): ILdapSyncTable {
+			const startedAt = new Date(sync.startedAt);
+			const endedAt = new Date(sync.endedAt);
+			const runTimeInMinutes = endedAt.getTime() - startedAt.getTime();
+			return {
+				runTime: humanizeDuration(runTimeInMinutes),
+				runMode: capitalizeFirstLetter(sync.runMode),
+				status: capitalizeFirstLetter(sync.status),
+				endedAt: convertToDisplayDate(endedAt.getTime()),
+				details: this.$locale.baseText('settings.ldap.usersScanned', {
+					interpolate: {
+						scanned: sync.scanned.toString(),
+					},
+				}),
+			};
+		},
+		async onSubmit(): Promise<void> {
+			// We want to save all form values (incl. the hidden onces), so we are using
+			// `values` data prop of the `FormInputs` child component since they are all preserved there
+			const formInputs = this.$refs.ldapConfigForm as Vue & { values: FormValues} | undefined;
+			if (!this.hasAnyChanges || !formInputs) {
+				return;
+			}
+
+			const newConfiguration: ILdapConfig = {
+				loginEnabled: formInputs.values.loginEnabled,
+				loginLabel: formInputs.values.loginLabel,
+				connectionUrl: formInputs.values.serverAddress,
+				allowUnauthorizedCerts: formInputs.values.allowUnauthorizedCerts,
+				connectionPort: formInputs.values.port,
```

Comment:
if you use a custom port, this is being passed in as `string`, which fails the validation on `PUT /rest/ldap/config`

Reply:
Yes, good catch. We were parsing this on the client side, but it looks like it got removed it at some point and since we cannot catch this with the tests I did not notice.